robotstxtpy/syntax.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import re
+from argparse import ArgumentParser
+# Check syntax
+
+def readText(filename):
```

Comment:
There is no need to read the text file as we could simply take an instance of `RobotsTxt` as the input.